index.js

```diff
@@ -79,8 +79,7 @@ function underPressure (fastify, opts, next) {
   }
 
   function sendError (res, next) {
-    res.statusCode = 503
-    res.setHeader('Retry-After', retryAfter)
+    res.status(503).header('Retry-After', retryAfter)
```

Comment:
Can you rename `res` in `reply`?

Reply:
Great! Can you also update the other occurrences?
Such as https://github.com/fastify/under-pressure/pull/23/files#diff-168726dbe96b3ce427e7fedce31bb0bcR62.